ecosystem/indexer-grpc/indexer-grpc-parser/README.md

```diff
@@ -0,0 +1,126 @@
+# Aptos Indexer
+
+Will fix readme but just remember command for now
+```
+APTOS_POSTGRES_CONNECTION_STRING_VAR=postgresql://localhost:5432/indexer_v2 APTOS_DATASTREAM_SERVICE_ADDRESS_VAR=http://localhost:50051 STARTING_VERSION=0 cargo run
+```
+
+
+> Tails the blockchain's transactions and pushes them into a postgres DB
+
+A fullnode can run an indexer with the proper configurations. If enabled, the indexer will tail
+transactions in the fullnode with business logic from  each registered `TransactionProcessor`. On
+startup, by default, the indexer will restart from the first gap (e.g. version 5 if versions succeeded are 0, 1, 2, 3, 4, 6). 
+
+Each `TransactionProcessor` will need to be run in a separate fullnode. Please note that it may be difficult to run several transaction processors simultaneously in a single machine due to port conflicts. 
+
+When developing your own, ensure each `TransactionProcessor` is idempotent, and being called with the same input will not result in an error if some or all of the processing had previously been completed.
+
+## Requirements
+
+- [Rust](https://rustup.rs/)
```

Comment:
@bowenyang007 The installation for Rust needs to match the one provided in this [script](https://github.com/aptos-labs/aptos-core/blob/main/scripts/dev_setup.sh), right?

Reply:
We'll have to revisit this as well lol. These instructions are just copied over from existing thing we'll clean up. Maybe it's good to just delete most of it and say that it's WIP.